include/seqan3/search/fm_index/fm_index.hpp

```diff
@@ -232,7 +232,11 @@ class fm_index
         // * sdsl construction currently only works for int_vector, std::string and char *, not ranges in general
         // uint8_t largest_char = 0;
         sdsl::int_vector<8> tmp_text(text.size());
-        ranges::copy(text | view::reverse | view::to_rank | view::transform([] (uint8_t const r) { return r + 1; }),
+
+        //TODO view::reverse is broken and can't be chained right now
```

Comment:
Is there an issue? If yes reference it, if no, make an issue and reference it :smile:

Reply:
I am already working on the issue...